projects/plugins/protect/src/js/components/admin-page/index.jsx

```diff
@@ -63,17 +64,18 @@ const ProtectAdminPage = () => {
 		return (
 			<AdminPage moduleName={ __( 'Jetpack Protect', 'jetpack-protect' ) } header={ <Logo /> }>
 				<AdminSectionHero>
-					<Container horizontalSpacing={ 3 } horizontalGap={ 7 }>
-						<Col sm={ 4 } md={ 4 } lg={ 6 }>
-							<AlertSVGIcon className={ styles[ 'alert-icon-wrapper' ] } />
-							<H3>{ __( 'We’re having problems scanning your site', 'jetpack-protect' ) }</H3>
-							<Text>{ displayErrorMessage }</Text>
-						</Col>
-						<Col sm={ 0 } md={ 0 } lg={ 1 }></Col>
-						<Col sm={ 4 } md={ 3 } lg={ 5 }>
-							<img src={ inProgressImage } alt="" />
-						</Col>
-					</Container>
+					<Dialog
+						primary={
+							<div className={ styles.primary }>
+								<AlertSVGIcon className={ styles[ 'alert-icon-wrapper' ] } />
+								<H3>{ __( 'We’re having problems scanning your site', 'jetpack-protect' ) }</H3>
+								<Text>{ displayErrorMessage }</Text>
+							</div>
+						}
+						secondary={ <img src={ inProgressImage } alt="" /> }
+						isTwoSections={ false }
+						isCard={ false }
+					/>
```

Comment:
Makes sense in terms of semantic this be a Dialog?

Feeling that we're using Dialog here only because the layout is based on two section side by side (we did the same at Footer).

But don't know if that means we should use Dialog, since don't like it's related with the initial scope and proposal of Dialog.

Reply:
> Makes sense in terms of semantic this be a Dialog?

This is very nice feedback. I'd say it doesn't make sense 

> Feeling that we're using Dialog here only because the layout is based on two section side by side (we did the same at Footer).

Also, the component takes over hiding the "secondary" section (the image) on mobile.

> But don't know if that means we should use Dialog, IMO this context is not related with the initial scope and proposal of Dialog.

I agree, but maybe it's a matter to rename the component. We have several similar cases that can be tackled by this component.